README.md

```diff
@@ -1 +1,49 @@
-# Portfolio-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse_Porfolio
+
+> This portfolio project is designed to showcase other projects undertaken by myself
+
+
+## Built With
+
+- HTML
+- CSS
+- Figma
+
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Kindly add the project repo link and give directions as to how one should clone and use it. 👍